### PR TITLE
Validate FUD

### PR DESCRIPTION
**Pull request by FuddlesNFT**
# Validate [{FUD}](https://solscan.io/token/{91Fv8PztVLsf4S49RANPGW92veJ2oL6BjwsP9i666Wxu})

## Attestations (Please provide links):
- Tweet from your Twitter Account attesting the Mint address, tagging [@JupiterExchange](https://twitter.com/JupiterExchange) and showing community support: https://twitter.com/FuddlesNFT/status/1731340243503092097}
- Coingecko/ CMC URL (If available): https://www.coingecko.com/en/coins/{id}

## Validation (Please check off boxes):
- [x] The metadata provided in the PR matches what is on-chain (Mandatory)
- [x] Does not duplicate the symbol of another token on Jupiter's strict list (If not, review will be delayed)
- [ ] Is Listed on Coingecko / CMC (Optional, but helpful for reviewers)  

## Acknowledgement (Please check off boxes)
- [x] My change matches the format in the file (no spaces between fields).
- [  ] My token is already live and trading on Jupiter.
- [x] !!! I read the README section on Community-Driven Validation and understand this PR will be only be reviewed when there is community support on Twitter.

### CODE REVIEW

File: validated-tokens.csv [hunk 1]
```diff
@@ -825,3 +825,4 @@ Ponk,PONK,HeqCcMjmuV5s25J49YiJyT6bD5qWLkP88YPajBySniaV,5,https://bafkreieyllpkeg
 ClownOnSolana,$Clown,V7ntWk4D9Frwaec2CUjMRMMsJG28Cd31HxDKNdP1qjM,9,https://arweave.net/q1EkX68UfbUnvjpA87GaRjGxVaP6f0lYMBeQXmFzN6w,true
 Solabrador,SOBER,2ZE6hSL36e44wP168YMnxrbi1CSCFuD2BJm7NoNHfsmN,9,https://node2.irys.xyz/5xOYKlM26hYXX5wkZoQhc1yw0Bd0Xzoh_SU52TakOZQ,true
 Solanacorn,CORN,6DSqVXg9WLTWgz6LACqxN757QdHe1sCqkUfojWmxWtok,7,https://arweave.net/c_Tbuf4_HSFuogvTV1_FaqqbQipml9YlAETM941s178,true
+Fuddles Shiba Inu,FUD,91Fv8PztVLsf4S49RANPGW92veJ2oL6BjwsP9i666Wxu,6,https://rl7lgv5motpbnrk57hjnjati2j2jxl2byu3lrtgpf7vgzppmrvxq.arweave.net/iv6zV6x03hbFXfnS1IJo0nSbr0HFNrjMzy_qbL3sjW8,true
```
Comment (Timoon21) [inline validated-tokens.csv:828]:
```suggestion
Fuddles Shiba Inu,FUD,91Fv8PztVLsf4S49RANPGW92veJ2oL6BjwsP9i666Wxu,6,https://arweave.net/iv6zV6x03hbFXfnS1IJo0nSbr0HFNrjMzy_qbL3sjW8,true
```

**Review by Timoon21 — DISMISSED** [review]

**Comment by 9yoi** [comment]
the tweet you listed on your description does not exist, i couldn't open the page

**Comment by FuddlesNFT** [comment]
@9yoi - https://twitter.com/FuddlesNFT/status/1731340243503092097

**Comment by FuddlesNFT** [comment]
@9yoi @Timoon21 @abhay @siong1987

We've updated our token tag from $FUD to $FUDSI and would like you to implement those changes to Jupiter: https://solscan.io/token/91Fv8PztVLsf4S49RANPGW92veJ2oL6BjwsP9i666Wxu

**Pull request merged**